### PR TITLE
fix tensorboard import issue

### PR DESCRIPTION
**Pull request by raubitsj**
Bug #691

### CODE REVIEW

File: wandb/tensorboard/watcher.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 for path in sys.path:
     if "client/wandb" in path:
         sys.path.remove(path)
+if sys.modules.get("tensorboard"):
+    del sys.modules["tensorboard"]
```
Comment (vanpelt) [inline wandb/tensorboard/watcher.py]: Whoa, nice find.

Comment (raubitsj) [reply wandb/tensorboard/watcher.py]: It was sitting in my tensorboard watcher WIP branch:
https://github.com/wandb/client/pull/585
Will try and get that revived and ready for review in the next couple weeks.

Comment (raubitsj) [reply wandb/tensorboard/watcher.py]: @vanpelt , can you take another look?  I had to do another workaround for `pip install ... #egg=wandb` python environments.

Comment (vanpelt) [reply wandb/tensorboard/watcher.py]: Cool, looks good.

**Pull request merged**